docs/static_site/src/pages/get_started/ubuntu_setup.md

```diff
@@ -147,67 +81,35 @@ On Ubuntu versions 16.04 or later, you need the following dependencies:
 **Step 1:** Install prerequisite packages.
 ```bash
     sudo apt-get update
-    sudo apt-get install -y build-essential git ninja-build ccache
-```
-
-**For Ubuntu 18.04 and CUDA builds you need to update CMake**
-
-```bash
-#!/usr/bin/env bash
-set -exuo pipefail
-sudo apt remove --purge --auto-remove cmake
-
-# Update CMAKE for correct cuda autotedetection: https://github.com/clab/dynet/issues/1457
-version=3.14
-build=0
-mkdir -p ~/tmp
-cd ~/tmp
-wget https://cmake.org/files/v$version/cmake-$version.$build.tar.gz
-tar -xzvf cmake-$version.$build.tar.gz
-cd cmake-$version.$build/
-./bootstrap
-make -j$(nproc)
-sudo make install
+    sudo apt-get install -y build-essential git ninja-build ccache python3-pip libopenblas-dev libopencv-dev
+    pip3 install --user --upgrade "cmake>=3.13.2"  # Instead of using pip, you could also manually install cmake from https://cmake.org
 ```
 
+Instead of `libopenblas-dev` you may also choose a different math library.
+Further information is provided in the source guide's [Math Library
+Selection](build_from_source#math-library-selection) section.
 
-**Step 2:** Install a Math Library.
+`libopencv-dev` is an optional dependency. You can delete it from above `apt-get
+install` line and build MXNet without OpenCV support by passing
+`-DUSE_OPENCV=OFF` to the `cmake` command below.
 
-Details on the different math libraries are found in the build from source guide's [Math Library Selection](build_from_source#math-library-selection) section.
-
-For OpenBLAS use:
-
-```bash
-    sudo apt-get install -y libopenblas-dev
-```
-
-For other libraries, visit the [Math Library Selection](build_from_source#math-library-selection) section.
-
-**Step 3:** Install OpenCV.
-
-*MXNet* uses [OpenCV](http://opencv.org/) for efficient image loading and augmentation operations.
-
-```bash
-    sudo apt-get install -y libopencv-dev
-```
-
-**Step 4:** Download MXNet sources and build MXNet core shared library.
-
-If building on CPU and using OpenBLAS:
+**Step 2:** Download MXNet sources
 
 Clone the repository:
 
 ```bash
-    git clone --recursive https://github.com/apache/incubator-mxnet.git
-    cd incubator-mxnet
+    git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet
+    cd mxnet
 ```
 
-Build with CMake and ninja, without GPU and without MKL.
+**Step 3:** Build MXNet core shared library.
+
+For a CPU-only build with OpenBLAS math library run:
 
 ```bash
     rm -rf build
     mkdir -p build && cd build
-    cmake -GNinja \
+    ~/.local/bin/cmake -GNinja \
```

Comment:
Somehow I found using this absolute path causes inconvenience as my 'cmake' is not installed on this path, nor is the default one on AWS DLAMI. Do you have a better idea to not using a fixed path?

Reply:
it would be better to just use CMake and specify what's the minimum required version

Reply:
@larroy some users may not know how to understand cmake.
@apeforest DLAMI encourages users to use conda, which causes problems for above installation steps.

I'll open a PR to improve the experience shortly